docs/installation.rst

```diff
@@ -17,3 +17,13 @@ Then run::
     pip -r requirements.txt
 
 as usual.
+
+Django
+------
+
+* Add 'bread' to your ``INSTALLED_APPS``
+* In any template where you're using Bread views, load bread's javascript
+  files, using something like::
+
+      <script src="{% static 'js/bread.js' %}"></script>
+      <script src="{% static 'js/URI.js' %}"></script>
```

Comment:
Not sure about this, but if bread.js uses URI.js, shouldn't URI.js be listed first?